p2p TestWriter flakes

```
=== RUN   TestWriter_timeout
=== RUN   TestWriter_timeout/NewWriter
=== RUN   TestWriter_timeout/NewWriter/WithContext
=== RUN   TestWriter_timeout/NewWriter/WithTimeout
##[error]    protobuf_test.go:266: timeout
=== RUN   TestWriter_timeout/NewWriterAndReader
=== RUN   TestWriter_timeout/NewWriterAndReader/WithContext
=== RUN   TestWriter_timeout/NewWriterAndReader/WithTimeout
--- FAIL: TestWriter_timeout (1.53s)
    --- FAIL: TestWriter_timeout/NewWriter (1.08s)
        --- PASS: TestWriter_timeout/NewWriter/WithContext (0.21s)
        --- FAIL: TestWriter_timeout/NewWriter/WithTimeout (0.87s)
    --- PASS: TestWriter_timeout/NewWriterAndReader (0.44s)
        --- PASS: TestWriter_timeout/NewWriterAndReader/WithContext (0.12s)
        --- PASS: TestWriter_timeout/NewWriterAndReader/WithTimeout (0.32s)
```

observed on the CI

more:
```
=== RUN   TestReader_timeout/NewWriterAndReader/WithContext
Error:     protobuf_test.go:113: got error <nil>, want context deadline exceeded
--- FAIL: TestReader_timeout (0.48s)
    --- PASS: TestReader_timeout/NewReader (0.18s)
        --- PASS: TestReader_timeout/NewReader/WithContext (0.18s)
    --- FAIL: TestReader_timeout/NewWriterAndReader (0.30s)
        --- FAIL: TestReader_timeout/NewWriterAndReader/WithContext (0.30s)
=== RUN   TestWriter
=== RUN   TestWriter/NewWriter
=== RUN   TestWriter/NewWriterAndReader
--- PASS: TestWriter (0.00s)
    --- PASS: TestWriter/NewWriter (0.00s)
    --- PASS: TestWriter/NewWriterAndReader (0.00s)
=== RUN   TestWriter_timeout
=== RUN   TestWriter_timeout/NewWriter
=== RUN   TestWriter_timeout/NewWriter/WithContext
=== RUN   TestWriter_timeout/NewWriterAndReader
=== RUN   TestWriter_timeout/NewWriterAndReader/WithContext
--- PASS: TestWriter_timeout (0.32s)
    --- PASS: TestWriter_timeout/NewWriter (0.18s)
        --- PASS: TestWriter_timeout/NewWriter/WithContext (0.18s)
    --- PASS: TestWriter_timeout/NewWriterAndReader (0.14s)
        --- PASS: TestWriter_timeout/NewWriterAndReader/WithContext (0.14s)
=== RUN   TestReadMessages
--- PASS: TestReadMessages (0.00s)
FAIL
FAIL	github.com/ethersphere/bee/pkg/p2p/protobuf	1.189s
```

And now `TestReader` flakes too
```
=== RUN   TestReader_timeout
=== RUN   TestReader_timeout/NewReader
=== RUN   TestReader_timeout/NewReader/WithContext
=== RUN   TestReader_timeout/NewWriterAndReader
=== RUN   TestReader_timeout/NewWriterAndReader/WithContext
Error:     protobuf_test.go:113: got error <nil>, want context deadline exceeded
--- FAIL: TestReader_timeout (0.48s)
    --- PASS: TestReader_timeout/NewReader (0.18s)
        --- PASS: TestReader_timeout/NewReader/WithContext (0.18s)
    --- FAIL: TestReader_timeout/NewWriterAndReader (0.30s)
        --- FAIL: TestReader_timeout/NewWriterAndReader/WithContext (0.30s)
=== RUN   TestWriter
=== RUN   TestWriter/NewWriter
=== RUN   TestWriter/NewWriterAndReader
--- PASS: TestWriter (0.00s)
    --- PASS: TestWriter/NewWriter (0.00s)
    --- PASS: TestWriter/NewWriterAndReader (0.00s)
=== RUN   TestWriter_timeout
=== RUN   TestWriter_timeout/NewWriter
=== RUN   TestWriter_timeout/NewWriter/WithContext
=== RUN   TestWriter_timeout/NewWriterAndReader
=== RUN   TestWriter_timeout/NewWriterAndReader/WithContext
--- PASS: TestWriter_timeout (0.32s)
    --- PASS: TestWriter_timeout/NewWriter (0.18s)
        --- PASS: TestWriter_timeout/NewWriter/WithContext (0.18s)
    --- PASS: TestWriter_timeout/NewWriterAndReader (0.14s)
        --- PASS: TestWriter_timeout/NewWriterAndReader/WithContext (0.14s)
=== RUN   TestReadMessages
--- PASS: TestReadMessages (0.00s)
FAIL
FAIL	github.com/ethersphere/bee/pkg/p2p/protobuf	1.189s
```

--- a/pkg/p2p/protobuf/protobuf_test.go
+++ b/pkg/p2p/protobuf/protobuf_test.go
@@ -75,7 +75,7 @@ func TestReader_timeout(t *testing.T) {
 		{
 			name: "NewReader",
 			readerFunc: func() protobuf.Reader {
-				return protobuf.NewReader(newMessageReader(messages, 100*time.Millisecond))
+				return protobuf.NewReader(newMessageReader(messages, 500*time.Millisecond))
 			},
 		},
 		{
@@ -83,7 +83,7 @@ func TestReader_timeout(t *testing.T) {
 			readerFunc: func() protobuf.Reader {
 				_, r := protobuf.NewWriterAndReader(
 					newNoopWriteCloser(
-						newMessageReader(messages, 100*time.Millisecond),
+						newMessageReader(messages, 500*time.Millisecond),
 					),
 				)
 				return r
@@ -97,7 +97,7 @@ func TestReader_timeout(t *testing.T) {
 				for i := 0; i < len(messages); i++ {
 					var timeout time.Duration
 					if i == 0 {
-						timeout = 600 * time.Millisecond
+						timeout = 1000 * time.Millisecond
 					} else {
 						timeout = 10 * time.Millisecond
 					}
@@ -176,14 +176,14 @@ func TestWriter_timeout(t *testing.T) {
 		{
 			name: "NewWriter",
 			writerFunc: func() (protobuf.Writer, <-chan string) {
-				w, msgs := newMessageWriter(100 * time.Millisecond)
+				w, msgs := newMessageWriter(500 * time.Millisecond)
 				return protobuf.NewWriter(w), msgs
 			},
 		},
 		{
 			name: "NewWriterAndReader",
 			writerFunc: func() (protobuf.Writer, <-chan string) {
-				w, msgs := newMessageWriter(100 * time.Millisecond)
+				w, msgs := newMessageWriter(500 * time.Millisecond)
 				writer, _ := protobuf.NewWriterAndReader(newNoopReadCloser(w))
 				return writer, msgs
 			},
@@ -196,7 +196,7 @@ func TestWriter_timeout(t *testing.T) {
 				for i, m := range messages {
 					var timeout time.Duration
 					if i == 0 {
-						timeout = 600 * time.Millisecond
+						timeout = 1000 * time.Millisecond
 					} else {
 						timeout = 10 * time.Millisecond
 					}

